vita3k/gui/src/settings_dialog.cpp

```diff
@@ -155,6 +155,15 @@ void draw_settings_dialog(GuiState &gui, HostState &host) {
     ImGui::PushStyleColor(ImGuiCol_Text, GUI_COLOR_TEXT_MENUBAR_OPTIONS);
     if (ImGui::BeginTabItem("GPU")) {
         ImGui::PopStyleColor();
+#ifdef USE_VULKAN
+        ImGui::Combo("Backend Render (Reboot for apply)", reinterpret_cast<int *>(&host.backend_renderer), "OpenGL\0Vulkan\0");
```

Comment:
Only change `Config`'s `backend_renderer` field. If you need a variable to hold the Combo's state, make one in `gui::State`.